opentaskspal/src/main/java/org/dmfs/opentaskspal/rowsets/Subtasks.java

```diff
@@ -35,10 +36,11 @@
  */
 public final class Subtasks extends DelegatingRowSet<TaskContract.Tasks>
 {
-
-    public Subtasks(String authority, ContentProviderClient client, @NonNull RowSnapshot<TaskContract.Tasks> parentTask)
+    public Subtasks(@NonNull String authority, @NonNull ContentProviderClient client, @NonNull Uri parentTask, String... projection)
     {
-        super(new QueryRowSet<>(new TasksView(authority, client), new ReferringTo<>(TaskContract.Tasks.PARENT_ID, parentTask)));
+        super(new QueryRowSet<>(
+                new TasksView(authority, client, projection),
```

Comment:
This `TaskView` should actually be passed to a ctor. There could be a secondary ctor which creates a default `TaskView` though.

Reply:
Done